src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,23 @@
-export const TodoList = () => {};
+import { TodoInfo } from '../TodoInfo';
+import { Todo } from '../../types/todo';
+import { User } from '../../types/user';
+
+type Props = {
+  todos: Todo[],
+  users: User[],
+};
+
+export const TodoList: React.FC<Props> = ({ todos, users }) => {
+  return (
+    <section className="TodoList">
+      {todos.map(todo => (
+        <TodoInfo
+          todo={todo}
+          users={users}
```

Comment:
Why do you need to pass all users to the TodoInfo? You should find the user before